lib/src/services/ds_dialog.service.dart

```diff
@@ -0,0 +1,153 @@
+import 'package:blip_ds/blip_ds.dart';
+import 'package:blip_ds/src/enums/ds_dialog_type.enum.dart';
+import 'package:blip_ds/src/widgets/buttons/ds_button.widget.dart';
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+
+/// A Design System's [Dialog] used to display a dialog box.
+class DSDialog {
+  final String title;
+  final String text;
+  final DSButton firstButton;
+  final DSButton? secondButton;
+  final BuildContext context;
+
+  /// Creates a new Design System's [Dialog]
+  DSDialog({
+    required this.title,
+    required this.text,
+    required this.context,
+    required this.firstButton,
+    this.secondButton,
+  });
+
+  void _show(final DSDialogType type) {
+    showDialog(
+      context: context,
+      barrierDismissible: false,
+      useSafeArea: true,
+      builder: (context) => _buildDialog(type),
+    );
+  }
+
+  Widget _buildDialog(final DSDialogType type) {
+    Widget buildHeader() {
+      final Color header = type == DSDialogType.system
+          ? DSColors.primaryLight
+          : type == DSDialogType.warning
+              ? DSColors.primaryYellowsCorn
+              : DSColors.extendRedsflower;
+
+      final String icon = 'assets/images/icon_alert_${describeEnum(type)}.png';
```

Comment:
My suggestion is to use svg for the icons. We will have less quality loss in size customizations because it is vector, and it is possible to customize the color when necessary.

Reply:
Good suggestion, but all the icons currently on the DS are in png, besides that we are studying to change the way we use it, so at the moment I think we can leave it like that